Directory.Build.props

```diff
@@ -21,8 +21,8 @@
          * Bump last two digits of the patch version for service releases.
          * Bump first digit of the patch version for feature releases (and reset the first two digits to 0)
     -->
-    <AndroidPackVersion>31.0.101</AndroidPackVersion>
```

Comment:
Maybe we should continue to use `.101` until we are consuming 6.0.200 .NET SDK builds? Alternatively, we might even want to stay on .101 until GA, and reserve `.200` for our second stable release? I am not sure what's best here...